tasks/es/ine.py

```diff
@@ -122,6 +123,9 @@ def populate(self):
                             output=self.output().table,
                             input=self.input()['data'].table))
 
+    def post_tasks(self):
```

Comment:
What is `post_tasks`? I can't see where this is called.

Reply:
Removed (it was from a previous attemp using dynamic dependencies)